src/lib/viewers/doc/_docBase.scss

```diff
@@ -1,36 +1,55 @@
 @import '../../boxuiVariables';
 
+$thumbnail-border-radius: 4px;
+
 .bp {
     .bp-thumbnails-container {
         border-right: solid 1px $seesee;
         display: flex;
-        flex: 0 0 180px;
+        flex: 0 0 181px;
     }
 
     .bp-thumbnail {
         align-items: center;
         background-color: #d8d8d8;
         border: 0;
-        border-radius: 4px;
+        border-radius: $thumbnail-border-radius;
         display: flex;
         flex: 1 1 auto;
         justify-content: center;
         margin: 0 15px;
-        overflow: hidden;
         padding: 0;
         position: relative;
     }
 
     .bp-thumbnail-page-number {
         background-color: $black;
-        border-radius: 4px;
+        border-radius: $thumbnail-border-radius;
         bottom: 10px;
         color: $white;
         font-size: 11px;
         line-height: 16px;
         opacity: .7;
         padding: 0 5px;
+        pointer-events: none;
+        position: absolute;
+    }
+
+    .bp-thumbnail-image {
+        border-radius: $thumbnail-border-radius;
+        pointer-events: none;
+        width: 100%;
+    }
+
+    .bp-thumbnail.bp-thumbnail-is-selected::after {
+        border: 4px solid $box-blue;
+        border-radius: $thumbnail-border-radius;
+        bottom: -2px;
```

Comment:
This smells a bit weird, is the size changing which requires these negative values?

Reply:
no this is a trick I found which allows me to put the selected border on the *outside* of the thumbnail button rather than taking the border width and height away from the thumbnail button itself. What I want to avoid is the thumbnail image resizing whenever the selection is applied or not

Reply:
cool, can you drop a comment explaining?

Reply:
will do